MIP104/MIP104.md

```diff
@@ -433,17 +433,17 @@ AllocatorDAOs must be actively market making Cash Stablecoins against Dai with a
 
 ##### 7.2.1.3: Cash Stablecoin Definition
 
-A Cash Stablecoin is a stablecoin listed in *7.2.1.3.1A* or a technically secure DeFi token that can be instantly converted to a listed stablecoin, or a custody solution that can be converted into a listed stablecoin within 30 minutes. The Stability Facilitators can modify *7.2.1.3.1A* through the weekly cycle to react to changing market conditions, opportunities or risks.
+A Cash Stablecoin is a stablecoin listed in *7.2.1.3.1A* or a technically secure DeFi token that can be instantly converted to a stablecoin listed in 7.2.1.3.1A , or a custody solution that can be converted into a listed stablecoin within 24 hours. The Stability Facilitators can modify *7.2.1.3.1A* through the weekly cycle to react to changing market conditions, opportunities or risks.
 
-###### 7.2.1.3.1A
+##### 7.2.1.3.1A
```

Comment:
This should be six #s since there are 5 numbers in the article number

```suggestion
###### 7.2.1.3.1A
```

Reply:
There's always one # more than numbers (because articles start after we've already used one heading level)